subworkflows/local/germline_variant_calling.nf

```diff
@@ -94,6 +96,17 @@ workflow GERMLINE_VARIANT_CALLING {
         haplotypecaller_vcf = CONCAT_HAPLOTYPECALLER.out.vcf
     }
 
+    if ('deepvariant' in params.tools.toLowerCase()) {
+
+        DEEPVARIANT(
+            bam,
```

Comment:
Based on the code for `bam` https://github.com/nf-core/sarek/blob/88ac395f1261d66c2bca6032f48d02ce69d44231/workflows/sarek.nf#L462   ... I am assuming that this is of the following shape 

```nextflow
    tuple val(meta), path(bam), path(bai)
```

Is this assumption correct?

Reply:
Totally